packages/experimental/RadioGroup/src/RadioGroup/useRadioGroup.ts

```diff
@@ -32,6 +32,7 @@ export const useRadioGroup = (props: RadioGroupProps): RadioGroupInfo => {
       accessible: true,
       accessibilityRole: 'radiogroup',
       accessibilityLabel: accessibilityLabel ?? label,
+      accessibilityState: { disabled: state.disabled },
```

Comment:
Should be memoized. You can use useMemo here.

Reply:
Fixed!